common/00_small_types_consolidated.cwt

```diff
@@ -184,6 +184,15 @@ faction = {
 	modifier = {
 		alias_name[modifier] = alias_match_left[modifier]
 	}
+
+	## cardinality = 0..1
+	triggered_faction_name = {
+		name = scalar
```

Comment:
In this case shouldn't it be "localisation" so it requires the right-side to be a localisation key ?

Akin to what estates have with custom_name:

```
custom_name = {
	desc = estate_ogboni
	trigger = {
		tag = OYO
	}
}
estate_ogboni:0 "Ogboni"
```